autogen/main/versions.tf.tmpl

```diff
@@ -46,13 +46,11 @@ terraform {
   required_providers {
     google = {
       source = "hashicorp/google"
-      # Workaround for https://github.com/hashicorp/terraform-provider-google/issues/19428
-      version = ">= 5.40.0, != 5.44.0, < 6.2.0, < 7"
+      version = ">= 5.44.1, < 7, !=6.0.0, !=6.0.1, !=6.1.0, !=6.2.0, !=6.3.0"
```

Comment:
could update these two to just be `> 6.4.0, <7` and then I _think_ this should work (but at the cost of making this (more) breaking)